packages/web-app/src/middleware/check-appeal-outcome.spec.js

```diff
@@ -32,10 +29,8 @@ describe('middleware/check-appeal-outcome', () => {
       req: {
```

Comment:
How does this test if the review oputcome is complete? check-appeal-outcome.js does not set any outcome anymore.

Reply:
They don't set the outcome but these tests are to ensure the middleware calls next when the _outcome_ is set

Reply:
OK